code/modules/research/techweb/all_nodes.dm

```diff
@@ -1,6 +1,6 @@
 
 //Current rate: 132500 research points in 90 minutes

```

Comment:
please make sure this is calculated to still be 90 minutes with only passive point generation and such.
as in, calculate how many points every single non-hidden unlockable node is, and then do the necessary math and set the generation rate in research subsystem to match.

Reply:
>and set the generation rate in research subsystem to match.

Alternatively, just delete this PR.

Reply:
There's only a net difference of 500 research points if I'm not mistaken. Is a difference of, what, 0.3% worth poking the algorithm for?

Reply:
I mean, the comment needs updating if the rate is changing

Reply:
Forcing it to always be 90 minutes seems unnecessary to me

Reply:
90 minute overall seems like a bad design. You could subvert this by just having a not very useful research node be super expensive, justifying raising the rate, and unlocking everything else faster.

Reply:
90 minute overall seems like a bad design. You could subvert this by just having a not very useful research node be super expensive, justifying raising the rate, and unlocking everything else faster.

Reply:
I agree with Shadowlight.